packages/tooling/fast-tooling/src/web-components/html-render-layer-inline-edit/html-render-layer-inline-edit.ts

```diff
@@ -63,9 +84,11 @@ export class HTMLRenderLayerInlineEdit extends HTMLRenderLayer {
 
     public handleTextInput(e: KeyboardEvent) {
         if (e.key === "Enter") {
-            this.commitEdit();
-            e.preventDefault();
-            return false;
+            if (this.commitmode === CommitMode.onEnter) {
```

Comment:
should this be included in the enclosing `if`? Combined with my change above you could use `if (e.key === "Enter" && this.commitMode.includes(CommitMode.onEnter))`

Reply:
You were right, but this condition has now been removed (see above comment).